src/test/groovy/readme/IncrementalExamples.java

```diff
@@ -0,0 +1,102 @@
+package readme;
+
+import graphql.Directives;
+import graphql.ExecutionInput;
+import graphql.ExecutionResult;
+import graphql.GraphQL;
+import graphql.incremental.DelayedIncrementalExecutionResult;
+import graphql.incremental.IncrementalExecutionResult;
+import graphql.schema.GraphQLSchema;
+import jakarta.servlet.http.HttpServletResponse;
+import org.reactivestreams.Publisher;
+import org.reactivestreams.Subscriber;
+import org.reactivestreams.Subscription;
+
+@SuppressWarnings({"unused", "ConstantConditions"})
+public class IncrementalExamples {
```

Comment:
I'm assuming these `*Examples` classes are used for documentation. Since everything we're doing is still experimental, I believe we don't need to worry about documenting it too much